crates/config/src/transform.rs

```diff
@@ -345,4 +356,22 @@ mod test {
     assert_eq!(actual, "screams");
     Ok(())
   }
+
+  #[test]
+  fn test_identifier_convention_convert() -> R {
+    // Other transformations are tested in string_conversion.rs
+    let trans = parse(
+      r#"
+      convert:
+        source: "$A"
+        identifier:
```

Comment:
One potentially confusing part is that `identifier` will not only convert identifiers but the whole matching metavariable text. Can we make it better?

Reply:
My idea was that you'd capture only the identifier you care about and convert it. How do you propose this could be improved? 🤔